pkg/interceptors/github/github.go

```diff
@@ -0,0 +1,64 @@
+package github
+
+import (
+	"errors"
+	"net/http"
+
+	gh "github.com/google/go-github/github"
+
+	"github.com/tektoncd/triggers/pkg/interceptors"
+
+	triggersv1 "github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+
+	"go.uber.org/zap"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+type Interceptor struct {
+	KubeClientSet kubernetes.Interface
+	Logger        *zap.SugaredLogger
+	Github        *triggersv1.GithubInterceptor
+}
+
+func NewInterceptor(gh *triggersv1.GithubInterceptor, k kubernetes.Interface, l *zap.SugaredLogger) interceptors.Interceptor {
+	return &Interceptor{
+		Logger:        l,
+		Github:        gh,
+		KubeClientSet: k,
+	}
+}
+
+func getSecretToken(cs kubernetes.Interface, gh *triggersv1.GithubInterceptor) ([]byte, error) {
+	ns := gh.SecretRef.Namespace
+	if ns == "" {
+		ns = metav1.NamespaceDefault
```

Comment:
Would it make sense to default to the namespace that the event listener is on? We do this in other places: https://github.com/tektoncd/triggers/blob/fccb70830e5836bce882513e357bafa283cb6baa/pkg/sink/sink.go#L154

Reply:
Done, thanks!